tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Ordering Party Get.feature

```diff
@@ -30,30 +31,36 @@ Scenario: 1. Get an ordering party
         | Fred      | Robinson | Fred.robinson@email.com | 12312543212     |
 
 @4616
-Scenario: 2. A non existent orderId returns not found
+Scenario: 2. Get an empty ordering party
+    When the user makes a request to retrieve the ordering-party section with the ID C000014-02
+    Then a response with status code 200 is returned
+    And an empty object is returned

```

Comment:
The line "And an empty object is returned" is more meaningful to a developer than a BA. What do you think to changing the phrase to a more business orientated phrase e.g. "And the response contains no data"

Reply:
👍